framework/src/task_manager/task_manager_register.c

```diff
@@ -73,3 +74,102 @@ int task_manager_register(char *name, int permission, int timeout)
 
 	return status;
 }
+
+int task_manager_register_task(char *name, int priority, int stack_size, main_t entry, char * argv[], int permission, int timeout)
```

Comment:
We already have task_manager_register API. Similar APIs names and its roles should be well-redefined.